common/src/main/java/com/wynntils/handlers/actionbar/ActionBarHandler.java

```diff
@@ -54,16 +52,11 @@ public void onActionBarUpdate(ChatPacketReceivedEvent.GameInfo event) {
         }
         previousRawContent = content;
 
-        Matcher matcher = content.getMatcher(ACTIONBAR_PATTERN);
-        if (!matcher.matches()) {
-            WynntilsMod.warn("ActionBarHandler pattern failed to match: " + content);
-            return;
-        }
+        StyledText[] segments = content.split("    "); // Wynn will provide at least 4 spaces between each segment
 
```

Comment:
Should we not do some kind of sanity check that we get exactly three segments? If we do not, we should either print a warning (and pad with empty segments if less than three, or drop/merge segments after the third), or just give up and return.

Reply:
We could, but I did a bunch of content and did not get any instance where actionbar crashed or had something other than 3 segments. I'll do it when I get home today, or tomorrow if I don't get time

Reply:
> at least 4 spaces

What if they send 8? we offset our math and cause confusion. Oh.. I just read magicus' comment. Yes, I wonder the same thing.

Reply:
Oh good point, I'll see if I can find an example